e2e/whisper/whisper_mailbox_test.go

```diff
@@ -117,6 +117,9 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 	s.Require().NoError(err)
 	s.Require().Nil(postResp.Err)
 
+	// @TODO(adam): if this line is uncommented, the test fails.
+	// time.Sleep(time.Second)
```

Comment:
@b00ris there is a race condition here. If I uncomment this timeout, a check below fails as it finds one message.